backends/libpurple/main.cpp

```diff
@@ -1440,6 +1440,7 @@ struct Dis {
 	std::string protocol;
 };
 
+__attribute__ ((unused))
```

Comment:
Looks like GCC-specific and will not work with Clang/MSVC

Reply:
This attribute is also [supported by Clang](https://clang.llvm.org/docs/AttributeReference.html#maybe-unused-unused-gnu-unused):

```
int main(int argc, char *argv[]) {
	return 0;
}

__attribute__ ((unused)) static int bla() {
	return 42;
}
```

```
$ clang -Wall test.cpp
```

MSVC seems to have problems with the attribute declaration.
But there is a simple workaround: https://stackoverflow.com/questions/19249521/attribute-macro-and-its-effect-on-visual-studio-2010-based-projects